icons/frame.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M7 4v16M17 4v16M4 7h16M4 17h16" />
```

Comment:
Can you split up the path in separate paths. Combined paths will sometimes render weirdly in  design programs.